tests/languages.rs

```diff
@@ -52,7 +52,7 @@ fn test_objective_c_auto_language_ambiguous_query() {
         "objective_c_project",
         r#"
             $ tree-sitter-grep -q '(identifier) @c'
-            File "./example.h" has ambiguous file-type, could be ObjectiveC, C, or Cpp. Try passing the --language flag
+            File "./example.h" has ambiguous file-type, could be C, Cpp, or ObjectiveC. Try passing the --language flag
```

Comment:
Ah interesting that this still prints "Cpp", I guess this is using the `#[derive(Debug)]` implementation on `SupportedLanguage`, does that support declarative "customizing"/"overriding" attributes?

Reply:
No, but strum does if you didn't mind using that: https://docs.rs/strum_macros/0.25.1/strum_macros/derive.Display.html (well technically `Display`, but that'd work I assume).

Or it seems pretty straightforward to add an implementation of `Debug` to your macro too?

Reply:
I like the `strum` solution especially given that (a) we're already hard-coding "unusual" `#[derive(...)]`'s in the macro (`clap::ValueEnum`) and (b) it's already set up to "flow those per-variant attributes through" to the generated enum definition, updated

Gave a couple of the other languages slightly more expected formatted names too eg "C#" or here "Objective-C"